### PR TITLE
[stable/cert-manager]: add pod labels

### PR DESCRIPTION
**Pull request by khrisrichardson**
**What this PR does / why we need it**:

* adds pod labels
* deduplicate fullname if name and release name are identical

### CODE REVIEW

File: stable/cert-manager/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cert-manager
-version: 0.2.10
+version: 0.2.11
```
Comment (timstoop) [inline stable/cert-manager/Chart.yaml]: You're adding a new feature, so that warrant a minor version bump instead of patch level.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
This chart is maintained upstream. Please create a PR there.
https://github.com/jetstack/cert-manager

**Pull request closed without merging**